src/style/app.module.css

```diff
@@ -6390,3 +6411,97 @@ button[data-testid='createPostBtn'] {
   font-weight: bold;
   color: var(--black-color);
 }
+
+/* people card */
+.personImage_peoplecard {
+  border-radius: 50%;
+  margin-right: 25px;
+  max-width: 70px;
+}
+
+.people_card_container {
+  display: flex;
+}
+
+.display_flex {
+  display: flex;
+}
+
+.align_center {
+  align-self: center;
+}
+
+.people_role {
+  width: 75%;
+  border: 1px solid #dee2e6;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  border-radius: 0.25rem;
+}
+
+/* People.tsx */
+
+.borderRounded8 {
+  border-radius: 8px;
+}
+
+.bottomRadius {
+  border-bottom-left-radius: 24px;
+  border-bottom-right-radius: 24px;
+}
+
+.shadow {
+  box-shadow: 5px 5px 4px 0px #d9d9d9;
+}
+
+.backgroundWhite {
+  background-color: white;
+}
+
+.people_content {
+  display: flex;
+  flex-direction: column;
+  height: fit-content;
+  min-height: calc(100% - 40px);
+}
+
+.gap {
+  gap: 20px;
+}
+
+.semiBold {
+  font-weight: 500;
+}
+
+.placeholderColor::placeholder {
+  color: #737373;
```

Comment:
Use a variable name not the hex code. Ensure that the variable name you choose describes the function of the color and not the color itself. This will help with future attempts to having Dark Mode.
- For example `--search-button-bg` is much better than `--light-blue`